Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -748,14 +772,29 @@ Common::Vec3 Wiimote::GetAcceleration()
   return accel;
 }
 
+Common::Vec3 Wiimote::GetAngularVelocity()
+{
+  auto orientation = Common::Matrix33::Identity();
+
+  // TODO: Make a function out of this:
```

Comment:
Just do it? ;)
For `Wiimote::GetAcceleration`'s sake.

Reply:
Okay, fine. :P